items/src/main/resources/application.conf

```diff
@@ -4,15 +4,21 @@ apm.secret=${?apm_secret}
 apm.service.name=${?apm_service_name}
 apm.environment=${?apm_environment}
 
+api.public-root="http://localhost"
+api.public-root=${?api_public_root}
+
+catalogue.api.publicRoot="https://api.wellcomecollection.org/catalogue/v2"
 catalogue.api.publicRoot=${?catalogue_api_public_root}
+
+content.api.publicRoot="https://api.wellcomecollection.org/content/v0"
 content.api.publicRoot=${?content_api_public_root}
 
 http.host="0.0.0.0"
+http.port=8080
```

Comment:
Do we actually use port and host separately?   Below, we have the default localhost url defined literally, rather than in terms of host and port.

Reply:
It's the other way round, we only use them separately AFAICT. In `externalBaseURL` the port gets trimmed off before it's used too so that is redundant.

Reply:
Agreed, the env vars & the way we construct the config here could probably be reconciled to something more sensible.